File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/05.While-Loop/02-exercise/01.oldBooks.js
function oldBooks(input) {

    const searchedBook = input[0];
    let index = 1;

    let current = input[index++];
    let bookCounter = 0;
    let bookIsFound = false;

    while (current !== "No More Books") {
        if (current === searchedBook) {
            bookIsFound = true;
            console.log(`You checked ${bookCounter} books and found it.`);
            break;
        }
        
        bookCounter += 1;
        current = input[index++];
    }

    if (!bookIsFound) {
        console.log("The book you search is not here!");
        console.log(`You checked ${bookCounter} books.`);
    }
}
oldBooks(["The Spot",
"Hunger Games",
"Harry Potter",
"Torronto",
"Spotify",
"No More Books"]);

File: Python_Basics/Homeworks-And-Labs/05.while_loop/02.exercise/01.old_books.py
searched_book = input()

current = input()
book_counter = 0
book_is_found = False

while current != "No More Books":
    if current == searched_book:
        book_is_found = True
        print(f'You checked {book_counter} books and found it.')
        break
        
    book_counter += 1
    current = input()

if not book_is_found:
    print("The book you search is not here!")
    print(f'You checked {book_counter} books.')
